DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/component_interfaces_comparison/streaming-invocation/Readme.md

```diff
@@ -0,0 +1,174 @@
+# Streaming Invocation and Conduit Arguments
+This implementation uses a streaming invocation interface, and conduits to pass the input pointer arguments instead of the CSR. It uses a single, shared memory-mapped host data interface.
+This design show how to specify conduit interfaces using `annotated_arg`.
+
+![](../assets/stream_kernel.svg)
+
+## Invocation Interface - Streaming pipelined interface
+In this design, the `SimpleVAdd` kernel uses a streaming pipelined invocation interface. This is explicitly specified using kernel properties. The following member function is added to the kernel functor to specify kernel properties:
+```cpp
+auto get(sycl::ext::oneapi::experimental::properties_tag) {
+    return sycl::ext::oneapi::experimental::properties{
+        sycl::ext::intel::experimental::streaming_interface<>,
+        sycl::ext::intel::experimental::pipelined<1>
+    };
+}
+```
+The property `sycl::ext::intel::experimental::streaming_interface<>` configures a streaming invocation interface with a `ready_in` interface to allow down-stream components to backpressure. You can choose to remove the `ready_in` interface by using `sycl::ext::intel::experimental::streaming_interface_remove_downstream_stall` instead. If you omit this property, the compiler will configure your kernel with a register-mapped invocation interface.
+
+The property `sycl::ext::intel::experimental::pipelined<1>` specifies that this streaming interface is pipelined with an II of 1. Other valid parameterization are:
```

Comment:
```suggestion
The property `sycl::ext::intel::experimental::pipelined<1>` specifies that this streaming interface is pipelined with an II of 1. Other valid parameterizations are:
```

Reply:
VS Code complains `parameterizations` is not a word